automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/NotebookInteractionSpec.scala

```diff
@@ -315,5 +315,22 @@ class NotebookInteractionSpec extends FreeSpec with LeonardoTestUtils with Befor
         }
       }
     }
+
+    "should install multiple user specified notebook extensions" in withWebDriver { implicit driver =>
+      val clusterName = ClusterName("user-jupyter-ext" + makeRandomId())
+      withNewCluster(billingProject, clusterName, ClusterRequest(Map(), None, None, None, Some(UserJupyterExtensionConfig(Map("translate"->testJupyterExtensionUri, "map"->"gmaps"),Map("jupyterlab"->"jupyterlab"), Map("pizza"->"pizzabutton"))))) { cluster =>
```

Comment:
Minor syntax comment: you can specify just a single argument like:
```
ClusterRequest(billingProject, clusterName, userJupyterExtensionConfig = Some(extensionConfig))
```
Might also be good to define the extension config in a separate variable so it's easier to see what's being installed.